client/components/domains/connect-domain-step/connect-domain-step-login.jsx

```diff
@@ -31,7 +29,7 @@ export default function ConnectDomainStepLogin( {
 			<p className={ className + '__text' }>
 				{ createInterpolateElement(
 					__(
-						'Log into your domain provider account (like GoDaddy, NameCheap, 1&1, etc.) If you can’t remember who this is: go to <a>this link</a>, enter your domain and look at <em>Reseller Information</em> or <em>Registrar</em> to see the name of your provider.'
+						'Log into your domain provider account (like GoDaddy, NameCheap, 1&1, etc.). If you can’t remember who this is: go to <a>this link</a>, enter your domain and look at <em>Reseller Information</em> or <em>Registrar</em> to see the name of your provider.'
```

Comment:
:ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.